modules/installation-aws-upi-bootstrap.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws_upi/installing-aws-upi.adoc
+
+[id="installation-aws-upi-bootstrap-{context}"]
+= Initializing the bootstrap node on AWS UPI
+
+After you create all of the required infrastructure in Amazon Web Services (AWS),
+you can install the cluster.
+
+.Prerequisites
+
+* Configure an AWS account.
+* Generate the ignition config files for your cluster.
+* Create and configure a VPC and assocated subnets in AWS.
+* Create and configure DNS, load balancers, and listeners in AWS.
+* Create control plane and worker roles.
+* Create the bootstrap node.
+* Create the control plane nodes.
+* If you plan to manually manage the worker machines, create the worker machines.
+
+.Procedure
+
+. Change to the directory that contains the installation program and run the
+following command:
++
+----
+$ ./openshift-install user-provided-infrastructure bootstrap-complete --dir=<installation-directory> <1>
```

Comment:
This is stale since openshift/installer#1552 with more changes in the pipe in openshift/installer#1568.  I'm not sure which version you're targeting in these docs; you may want to wait until openshift/installer#1568 settles and then pivot to wherever it ended up.

nit: I also prefer setting `--dir` right after `openshift-install` to help underline its position as an `openshift-install` option.  You *can* set it later, but I don't want users stopping to think "does this other subcommand support `--dir` or not?", because you can always set it.

Reply:
@wking, we're targeting beta4 for this drop. If you expect openshift/installer#1568 to land before that drop, I'll pivot to that.

Reply:
"bootstrap-complete" won

Reply:
> If you expect openshift/installer#1568 to land...

We've abandoned that one :p, so you should be clear to adapt this line to the already-landed e openshift/installer#1552 now.